tox.ini

```diff
@@ -0,0 +1,67 @@
+[tox]
+envlist = flake8,pydocstyle,docs
+
+
+[testenv]
+basepython=python2.7
+# Installing with sdist makes tox/pip very unhappy.
+usedevelop=True
+# Bodhi requires koji, which isn't available on PyPI.
+sitepackages=True
+
+
+[testenv:docs]
+changedir = docs
+deps =
+    sphinx
+    cornice_sphinx
+whitelist_externals =
+    mkdir
+    rm
+    sphinx-build
+commands=
+    rm -rf _build images
+    mkdir -p _static images
+    ./generate_db_schema
+    sphinx-build -W -b html -d {envtmpdir}/doctrees .  _build/html
+    sphinx-build -W -b man -d {envtmpdir}/doctrees .  _build/man
```

Comment:
I was having trouble with the docs environment in tox, but I'm happy to poke at that in a separate PR